### ISSUE TITLE
No access for Portis-Bright ID users

### ISSUE THREAD

**Issue by jessicazartler** [1Hive/celeste-dashboard#51]
I am going through testing Celeste and wondering if I can change the wallet that my BrightID is associated with for 1Hive? Because of a bug, I signed up with Portis instead of Metamask for Bright ID/faucet -- and it's a real pain - and noticing that in Celeste, there is only Metamask/Frame - so all of the people who used Portis will not be able to become jurors it seems?

Is it possible to change the BrightID/wallet association - is it 1Hive side or Bright ID side? There is no option for this in Bright ID app, so guessing it is 1Hive issue..

Thank you for all the hard work! The UI/UX/design looks great!

**Comment by fabriziovigevani** [comment]
Hey Jessica!

If you go to the [Faucet](https://faucet.1hive.org/#/) and do the same process of scanning the qr code while being connected with a different address, that address is going to override the current associated address with your BrightId account.

Let me know if you encounter any other issue.

**Comment by jessicazartler** [comment]
Hi Fabrizio, thanks so much for your message and getting back! I know you guys are super busy!

I signed in with MetaMask and got this error - I never was able to access the QR with MetaMask, which is why I got the Portis account:

METAMASK VIEW
<img width="356" alt="error" src="https://user-images.githubusercontent.com/43972522/110372447-4df0d300-801c-11eb-8da4-c4af05aa578a.png">

The Portis view as well in case it's helpful:

PORTIS VIEW
<img width="596" alt="portis view" src="https://user-images.githubusercontent.com/43972522/110372487-5517e100-801c-11eb-81c5-afdec73a614c.png">

*I should note that I did import my Portis wallet to my Metamask, I tried to activate celeste from that account and it is still saying I am not verified by Bright ID..

Thanks for your help! Just wanted to flag it as well for other Portis users as there is no option on Celeste for Portis people
Keep up the great work! :D

**Comment by rperez89** [comment]
> Hi Fabrizio, thanks so much for your message and getting back! I know you guys are super busy!
> 
> I signed in with MetaMask and got this error - I never was able to access the QR with MetaMask, which is why I got the Portis account:
> 
> METAMASK VIEW
> <img alt="error" width="356" src="https://user-images.githubusercontent.com/43972522/110372447-4df0d300-801c-11eb-8da4-c4af05aa578a.png">
> 
> The Portis view as well in case it's helpful:
> 
> PORTIS VIEW
> <img alt="portis view" width="596" src="https://user-images.githubusercontent.com/43972522/110372487-5517e100-801c-11eb-81c5-afdec73a614c.png">
> 
> *I should note that I did import my Portis wallet to my Metamask, I tried to activate celeste from that account and it is still saying I am not verified by Bright ID..
> 
> Thanks for your help! Just wanted to flag it as well for other Portis users as there is no option on Celeste for Portis people
> Keep up the great work! :D

hey jessica! we are going to disable the portis wallet provider given that is not working great, but wondering if you can use another account that has nothing to see with portis? and try to associate that one on the faucet?  if you already tryied that are you still seeing that error without even the QR code shows?

**Comment by fabriziovigevani** [comment]
Yes she is getting the `The user is sponsored before` error when trying to associate a different account through metamask, maybe this issue is due to trying to sponsor an individual already sponsored and we should skip it.

**Comment by rperez89** [comment]
> Yes she is getting the `The user is sponsored before` error when trying to associate a different account through metamask, maybe this issue is due to trying to sponsor an individual already sponsored and we should skip it.

yeah but given what she wrote here `I should note that I did import my Portis wallet to my Metamask,` seems that she is trying to use the same account right?
